db/db.go

```diff
@@ -2239,22 +2241,39 @@ func GetAddressWithdrawals(address []byte, limit uint64, pageToken string) ([]*t
 	FROM blocks_withdrawals w
 	INNER JOIN blocks b ON b.blockroot = w.block_root AND b.status = '1'
 	WHERE w.address = $1 AND w.withdrawalindex <= $2
-	ORDER BY w.withdrawalindex DESC LIMIT $3`, address, withdrawalindex, limit+1)
+	ORDER BY w.withdrawalindex DESC LIMIT $3`, address, withdrawalIndex, limit+1)
 	if err != nil {
-		if err == sql.ErrNoRows {
-			return withdrawals, "", nil
+		if err != sql.ErrNoRows {
+			return nil, fmt.Errorf("error getting blocks_withdrawals for address: %x: %w", address, err)
+		}
+		nextPageToken = ""
+	} else {
+		// Get the next page token and remove that withdrawal from the results
+		nextPageToken = endOfWithdrawalsData
+		if len(withdrawals) == int(limit+1) {
+			nextPageToken = fmt.Sprintf("%d", withdrawals[limit].Index)
+			withdrawals = withdrawals[:limit]
+		}
+	}
+
+	withdrawalsData := make([][]interface{}, len(withdrawals))
+	for i, w := range withdrawals {
+		withdrawalsData[i] = []interface{}{
+			template.HTML(fmt.Sprintf("%v", utils.FormatEpoch(utils.EpochOfSlot(w.Slot)))),
+			template.HTML(fmt.Sprintf("%v", utils.FormatBlockSlot(w.Slot))),
+			template.HTML(fmt.Sprintf("%v", utils.FormatTimestamp(utils.SlotToTime(w.Slot).Unix()))),
+			template.HTML(fmt.Sprintf("%v", utils.FormatValidator(w.ValidatorIndex))),
+			template.HTML(utils.FormatClCurrency(w.Amount, currency, 6, true, false, false, true)),
 		}
-		return nil, "", fmt.Errorf("error getting blocks_withdrawals for address: %x: %w", address, err)
 	}
 
-	// Get the next page token and remove that withdrawal from the results
-	nextPageToken := endOfWithdrawalsData
-	if len(withdrawals) == int(limit+1) {
-		nextPageToken = fmt.Sprintf("%d", withdrawals[limit].Index)
-		withdrawals = withdrawals[:limit]
+	resultData := &types.DataTableResponse{
```

Comment:
**nitpick:** `resultData` could be called `data` to be consistent with the other functions.

Reply:
done